src/interfaces/wasm_bindgen/hybrid_cc_aes.rs

```diff
@@ -294,5 +299,17 @@ pub fn webassembly_hybrid_decrypt(
         )
         .map_err(|e| JsValue::from_str(&format!("Error decrypting ciphertext: {e}")))?;
 
-    Ok(Uint8Array::from(cleartext.as_slice()))
+    let mut result = vec![];
```

Comment:
You should use the serializer from CryptoCore. Something like:
``` rust
let ser = Serializer::new();
ser.write_vec(cleartext_header.additional_data.as_slice())?;
ser.write_array(cleartext.as_slice())?;
Ok(Uint8Array::from(ser.value()))
```

Reply:
Done in 2937cfe